templates/pulpcore-resource-manager.service.erb

```diff
@@ -11,10 +11,14 @@ User=<%= scope['pulpcore::user'] %>
 Group=<%= scope['pulpcore::group'] %>
 WorkingDirectory=<%= scope['pulpcore::user_home'] %>
 RuntimeDirectory=pulpcore-resource-manager
+<% if scope['pulpcore::use_legacy_tasking_system'] %>
```

Comment:
https://github.com/theforeman/puppet-pulpcore/pull/202 already takes care of this. Also, if you're using lines like this, you should strip trailing whitespace to avoid empty lines
```suggestion
<% if scope['pulpcore::use_legacy_tasking_system'] -%>
```